twoinputs/demand-windowsize.ipynb

```diff
@@ -0,0 +1,831 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# UIUC Grid Demand: Demand + Solar Elevation Angle\n",
+    "\n",
+    "This notebook uses an Echo State Network(ESN) (from pyESN) to predict the demand at the University of Illinois.\n",
+    "Rather than one input, there are two inputs: Demand and Solar Elevation Angle. \n",
+    "The following items are included:\n",
+    "\n",
+    "1. An initial demonstration of the ESN with a 72-hour prediction.\n",
+    "2. Grid optimization over spectral radius and noise\n",
+    "3. Grid optimization over reservoir size and sparsity\n",
+    "4. Demonstrate the effect of training length on accuracy.\n",
+    "5. A final demonstration of the ESN with optimized hyper-parameters."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Windows\n"
+     ]
+    }
+   ],
+   "source": [
+    "import platform \n",
+    "operating_system = platform.system()\n",
+    "print(operating_system)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import time\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import matplotlib.pyplot as plt\n",
+    "%matplotlib inline\n",
+    "\n",
+    "import sys\n",
+    "if operating_system is 'Linux':\n",
+    "    sys.path.insert(0, \"/home/dotson/research/cairo\")\n",
+    "elif operating_system is 'Windows':\n",
+    "    sys.path.insert(0, \"C:/users/samgd/research/cairo\")\n",
```

Comment:
Depending on how you want to use this code, consider changing how the paths are implemented.